script/utils/update-redirects-data.ts

```diff
@@ -23,9 +32,9 @@ const updateRedirectsData = async(currentBook: BookWithOSWebData, newBook: BookW
   const flatCurrentTree = flattenArchiveTree(currentBook.tree).filter((section) => section.id !== currentBook.id);
   const flatNewTree = flattenArchiveTree(newBook.tree).filter((section) => section.id !== currentBook.id);
```

Comment:
Idk what exactly this filter is for, but in this case I think the I'd might be wrong